Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:6
```

Comment:
another option, which we do in calc, is to leave it hard coded but add a unit test that verifies the Dockerfile is in sync with .nvmrc.